_posts/2020-08-28-accesibility-in-mind.md

```diff
@@ -0,0 +1,74 @@
+
+## 4. It improves content quality
+
+Navigating your UI with a screen reader is an eye-opening experience which no front-end developer should spare themselves. The screen reader makes unnecessary repetition painfully obvious.
+
+Lack of clarity in the titles and the labels also sticks out. Uncovering lack of clarity in these crucial pieces of text is of tremendous importance because two pitfalls await the unawary designer: First, the full visual experience often compensates for lack of clarity with context. Danger lurks here because this supporting context might change or even go away. Secondly, the designer is too close to their creation, too close to the business concepts and too involved in the development process to identify lack of clarity. The "fresh eyes" a designer needs to view their UI with is just what a screen reader provides.
+
+Screen Readers help weed out weaknesses in navigation, too. Listening to your document flow might make your rethink the order in which you present your content. At the very least, it's a useful tool for testing your assumptions about content layout undistracted.
+
+## 5. It makes your app adjustable to user’s preferences and needs
+
+Providing a value for the [`lang`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#attr-lang) attribute on the `<html>` element is optional. However, WCAG [3.1.1 - Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page) marks it as required. The integrated translation feature on the Google Chrome browser requires lang to be properly set. Modern web apps target a global audience and Google Chrome owns majority market share. Therefore, it makes great business sense to follow this guideline. At the end of the day, it is a testament to how the WACG may serve as a best-practices document for web development.
+Furthermore, WCAG provides rules about content visibility (for instance [1.4.4 - Resize text:](https://www.w3.org/TR/WCAG21/#resize-text) *“text can be resized without assistive technology up to 200 percent without loss of content or functionality”* and [1.4.10 - Reflow](https://www.w3.org/TR/WCAG21/#reflow), which sets a screen limit for scrolling in two dimensions). Following these guidelines provides advanced UX experience for users who do not really need accessibility, but they choose to zoom the app through browser settings or work on a small window or on a mobile device.
```

Comment:
"advanced UX experience" -> "**an** advanced/improved user experience" ("UX experience" is a bit like "ATM machine" :P)

"..., but **who** choose"

Reply:
Done